command/mount.go

```diff
@@ -54,6 +56,7 @@ func (c *MountCommand) Run(args []string) int {
 			DefaultLeaseTTL: defaultLeaseTTL,
 			MaxLeaseTTL:     maxLeaseTTL,
 		},
+		Local: local,
```

Comment:
I think you mean "always false" :-) But no -- if they switch later the behavior will still be what was set at mount time.